### PR TITLE
Minor updates to Access Review Beta

### PR DESCRIPTION
**Pull request by shubhamguptacal**
Super minor updates to Access Review Beta to include configuration and surfacing up of insights

Schema update PR: 5258874

### CODE REVIEW

File: api-reference/beta/resources/enums.md [hunk 1]
```diff
@@ -2617,3 +2617,11 @@ Possible values for user account types (group membership), per Windows definitio
...
 |other|
 |unknownFutureValue|
+
+### userSignInRecommendationScope values 
+
+|Member|
+|:---|
+|tenant|
+|application|
+|unknownFutureValue|
```
Comment (isabelleatmsft) [inline api-reference/beta/resources/enums.md:2627]: @shubhamguptacal is the unknownFutureValue supposed to be there?

Comment (shubhamguptacal) [reply api-reference/beta/resources/enums.md:2627]: Yes, this is standard for all enums

**Review by isabelleatmsft — APPROVED** [review]

File: api-reference/beta/resources/accessreviewrecommendationinsightsetting.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+## Relationships
+None.
+
+## JSON representation
+The following is a JSON representation of the resource.
+<!-- {
+  "blockType": "resource",
+  "@odata.type": "microsoft.graph.accessreviewrecommendationinsightsetting"
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/accessreviewrecommendationinsightsetting.md:28]:
```suggestion
  "blockType": "resource",
  "@odata.type": "microsoft.graph.accessReviewRecommendationInsightSetting"
```

File: api-reference/beta/resources/accessreviewrecommendationinsightsetting.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+}
+-->
+``` json
+{
+  "@odata.type": "#microsoft.graph.accessreviewrecommendationinsightsetting",
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/accessreviewrecommendationinsightsetting.md:33]:
```suggestion
  "@odata.type": "#microsoft.graph.accessReviewRecommendationInsightSetting",
```

File: api-reference/beta/resources/accessreviewrecommendationinsightsetting.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+title: "accessReviewRecommendationInsightSetting resource type"
+description: "In the Azure AD access reviews feature, the `accessreviewrecommendationinsightsetting` represents the settings associated with an insight included in the review."
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/accessreviewrecommendationinsightsetting.md:3]:
```suggestion
description: "In the Azure AD access reviews, the accessReviewRecommendationInsightSetting represents the settings associated with an insight provided for an access review."
```

File: api-reference/beta/resources/accessreviewschedulesettings.md [hunk 5]
```diff
@@ -31,6 +31,7 @@ The **accessReviewScheduleSettings** defines the settings of an [accessReviewSch
...
 | recommendationsEnabled|Boolean | Indicates whether decision recommendations are enabled or disabled. |
 | recommendationLookBackDuration | Duration| Optional field. Indicates the time period of inactivity (with respect to the start date of the review instance) that recommendations will be configured from. The recommendation will be to `deny` if the user is inactive during the look back duration. For reviews of groups and Azure AD roles, any duration is accepted. For reviews of applications, 30 days is the maximum duration. If not specified, the duration is 30 days. |
+| recommendationInsightSettings|[accessReviewRecommendationInsightSetting](../resources/accessReviewRecommendationInsightSetting.md) collection | Optional field. Describes the types of insights that aid reviewers making decisions to include in the review. There is one type that is currently supported: `userLastSignInRecommendationInsightSetting` (default) |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/accessreviewschedulesettings.md:34]: Too "early" for the statement on the supported derived type. That'll be in the accessReviewRecommendationInsightSetting resource description.
```suggestion
| recommendationInsightSettings|[accessReviewRecommendationInsightSetting](../resources/accessReviewRecommendationInsightSetting.md) collection | Optional. Describes the types of insights that aid reviewers to make access review decisions. |
```

File: api-reference/beta/resources/userlastsignInrecommendationinsightsetting.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+# userlastsignInrecommendationinsightsetting resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]
+
+The **userlastsignInrecommendationinsightsetting** allows you to configure last sign-in as an insight to aid reviewer decision-making for an [accessReviewScheduleDefinition](accessreviewscheduledefinition.md).
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/userlastsignInrecommendationinsightsetting.md:18]:
```suggestion
The **userLastSignInRecommendationInsightSetting** allows you to configure the last sign-in date and time of a user as an insight to aid the reviewers to make decisions for an [accessReviewScheduleDefinition](accessreviewscheduledefinition.md) object.

Inherits from [accessReviewRecommendationInsightSetting](accessReviewRecommendationInsightSetting.md).
```

File: api-reference/beta/resources/userlastsignInrecommendationinsightsetting.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+---
+title: "userlastsigninrecommendationinsightsetting resource type"
+description: "In the Azure AD access reviews feature, the `userlastsignInrecommendationinsightsetting` represents the settings associated with the last sign-in based insight included in the review, to aid reviewer decision-making."
+author: "shubhamguptacal"
+ms.localizationpriority: medium
+ms.prod: "governance"
+doc_type: resourcePageType
+---
+
+# userlastsignInrecommendationinsightsetting resource type
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/userlastsignInrecommendationinsightsetting.md:10]: Same case as in the schema
```suggestion
title: "userLastSignInRecommendationInsightSetting resource type"
description: "In the Azure AD access reviews, the userLastSignInRecommendationInsightSetting represents the settings associated with the insight that is based on the last sign-in date and time of the user, and that is used to aid reviewers to make decisions."
author: "shubhamguptacal"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# userlastsignInrecommendationinsightsetting resource type
```

File: api-reference/beta/resources/userlastsignInrecommendationinsightsetting.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]
+
+The **userlastsignInrecommendationinsightsetting** allows you to configure last sign-in as an insight to aid reviewer decision-making for an [accessReviewScheduleDefinition](accessreviewscheduledefinition.md).
+
+## Properties
+| Property    | Type   | Description |
+| :---------------| :---------- | :---------- |
+| signInScope| userSignInRecommendationScope | Indicates whether inactivity is calculated based on activity in the tenant or in the application. Application is only relevant when the access review is a review of assignment to an application. |
+| recommendationLookBackDuration | Duration| Optional field. Indicates the time period of inactivity (with respect to the start date of the review instance) that recommendations will be configured from. The recommendation will be to `deny` if the user is inactive during the look back duration. For reviews of groups and Azure AD roles, any duration is accepted. For reviews of applications, 30 days is the maximum duration. If not specified, the duration is 30 days. |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/userlastsignInrecommendationinsightsetting.md:24]: Alphabetically order properties in the table
```suggestion
| recommendationLookBackDuration | Duration | Optional. Indicates the time period of inactivity (with respect to the start date of the review instance) that recommendations will be configured from. The recommendation will be to `deny` if the user is inactive during the look-back duration. For reviews of groups and Azure AD roles, any duration is accepted. For reviews of applications, 30 days is the maximum duration. If not specified, the duration is 30 days. |
| signInScope | userSignInRecommendationScope | Indicates whether inactivity is calculated based on the user's inactivity in the tenant or in the application. The possible values are `tenant`, `application`, `unknownFutureValue`. `application` is only relevant when the access review is a review of an assignment to an application. |
```

File: api-reference/beta/resources/userlastsignInrecommendationinsightsetting.md [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+## Relationships
+None.
+
+## JSON representation
+The following is a JSON representation of the resource.
+<!-- {
+  "blockType": "resource",
+  "@odata.type": "microsoft.graph.userlastsignInrecommendationinsightsetting"
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/userlastsignInrecommendationinsightsetting.md:33]:
```suggestion
  "blockType": "resource",
  "@odata.type": "microsoft.graph.userLastSignInRecommendationInsightSetting",
  "baseType": "microsoft.graph.accessReviewRecommendationInsightSetting"
```

File: api-reference/beta/resources/userlastsignInrecommendationinsightsetting.md [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+}
+-->
+``` json
+{
+  "@odata.type": "#microsoft.graph.userlastsignInrecommendationinsightsetting",
+  "recommendationLookBackDuration": "Duration",
+  "signInScope": "microsoft.graph.userSignInRecommendationScope"
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/userlastsignInrecommendationinsightsetting.md:40]: The cause of one of the API doctor errors
```suggestion
  "signInScope": "String"
```

File: api-reference/beta/resources/governanceinsight.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
+---
+title: "governanceinsight resource type"
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/governanceinsight.md:2]:
```suggestion
title: "governanceInsight resource type"
```

File: api-reference/beta/resources/governanceinsight.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+description: "In the Azure AD access reviews feature, the `governanceinsight` represents a type of insight included in the review to aid reviewer decision making."
+author: "shubhamguptacal"
+ms.localizationpriority: medium
+ms.prod: "governance"
+doc_type: resourcePageType
+---
+
+# governanceinsight resource type
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/governanceinsight.md:10]:
```suggestion
# governanceInsight resource type
```

File: api-reference/beta/resources/enums.md [hunk 13]
```diff
@@ -2617,3 +2617,11 @@ Possible values for user account types (group membership), per Windows definitio
 |faculty|
 |other|
 |unknownFutureValue|
+
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/enums.md:2620]: Please move this entry to first in the file

File: api-reference/beta/resources/accessreviewschedulesettings.md [hunk 14]
```diff
@@ -61,7 +62,12 @@ The following is a JSON representation of the resource.
     }
   ],
   "recommendationsEnabled": "Boolean",
-  "recommendationLookBackDuration": "Duration"
+  "recommendationLookBackDuration": "Duration",
+  "recommendationInsightSettings": [
+    {
+      "@odata.type": "microsoft.graph.accessReviewRecommendationInsightSetting"
+    }
+  ],
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/accessreviewschedulesettings.md:70]:
```suggestion
  ]
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 15]
```diff
@@ -627,6 +627,72 @@
...
+              "ChangeType": "Addition",
+              "Description": "Added the [accessReviewRecommendationInsightSetting](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewRecommendationInsightSetting?view=graph-rest-beta) resource type.",
+              "Target": "accessReviewRecommendationInsightSetting"
+            },
+            {
+              "Id": "e96e9533-1955-47a6-92ed-3e9a45e87e1f",
+              "ApiChange": "Property",
+              "ChangedApiName": "recommendationInsightSettings",
+              "ChangeType": "Addition",
+              "Description": "Added the **recommendationInsightSettings** property to [accessReviewScheduleSettings](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewScheduleSettings?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:654]:
```suggestion
              "Description": "Added the **recommendationInsightSettings** property to [accessReviewScheduleSettings](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewScheduleSettings?view=graph-rest-beta) resource. Use this property to configure insights that aid reviewers of an access review to make decisions.",
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 16]
```diff
@@ -627,6 +627,72 @@
...
+              "ChangeType": "Addition",
+              "Description": "Added the [userLastSignInRecommendationInsightSetting](https://docs.microsoft.com/en-us/graph/api/resources/userLastSignInRecommendationInsightSetting?view=graph-rest-beta) resource type.",
+              "Target": "userLastSignInRecommendationInsightSetting"
+            },
+            {
+              "Id": "e96e9533-1955-47a6-92ed-3e9a45e87e1f",
+              "ApiChange": "Resource",
+              "ChangedApiName": "insights",
+              "ChangeType": "Addition",
+              "Description": "Added the **insights** relationship to [accessReviewInstanceDecisionItem](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItem?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:670]:
```suggestion
              "Description": "Added the **insights** navigation property to the [accessReviewInstanceDecisionItem](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItem?view=graph-rest-beta) resource.",
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 17]
```diff
@@ -627,6 +627,72 @@
...
+              "ChangeType": "Addition",
+              "Description": "Added the **recommendationInsightSettings** property to [accessReviewScheduleSettings](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewScheduleSettings?view=graph-rest-beta) resource.",
+              "Target": "accessReviewScheduleSettings"
+            },
+            {
+              "Id": "e96e9533-1955-47a6-92ed-3e9a45e87e1f",
+              "ApiChange": "Resource",
+              "ChangedApiName": "userLastSignInRecommendationInsightSetting",
+              "ChangeType": "Addition",
+              "Description": "Added the [userLastSignInRecommendationInsightSetting](https://docs.microsoft.com/en-us/graph/api/resources/userLastSignInRecommendationInsightSetting?view=graph-rest-beta) resource type.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:662]:
```suggestion
              "Description": "Added the [userLastSignInRecommendationInsightSetting](https://docs.microsoft.com/en-us/graph/api/resources/userLastSignInRecommendationInsightSetting?view=graph-rest-beta) resource type. Use this type to configure insight settings based on the user's last sign-in date and time into a tenant or application.",
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 18]
```diff
@@ -627,6 +627,72 @@
...
+              "ChangeType": "Addition",
+              "Description": "Added the **userSignInRecommendationScope** enumeration type.",
+              "Target": "userSignInRecommendationScope"
+            },
+            {
+              "Id": "e96e9533-1955-47a6-92ed-3e9a45e87e1f",
+              "ApiChange": "Resource",
+              "ChangedApiName": "accessReviewRecommendationInsightSetting",
+              "ChangeType": "Addition",
+              "Description": "Added the [accessReviewRecommendationInsightSetting](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewRecommendationInsightSetting?view=graph-rest-beta) resource type.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:646]:
```suggestion
              "Description": "Added the [accessReviewRecommendationInsightSetting](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewRecommendationInsightSetting?view=graph-rest-beta) resource type. This is an abstract type that exposes resources for configuring the type of insights to provide to reviewers and access review decision-makers.",
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 19]
```diff
@@ -627,6 +627,72 @@
...
+              "ChangeType": "Addition",
+              "Description": "Added the **insights** relationship to [accessReviewInstanceDecisionItem](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItem?view=graph-rest-beta) resource.",
+              "Target": "accessReviewInstanceDecisionItem"
+            },
+            {
+              "Id": "e96e9533-1955-47a6-92ed-3e9a45e87e1f",
+              "ApiChange": "Resource",
+              "ChangedApiName": "governanceInsight",
+              "ChangeType": "Addition",
+              "Description": "Added the [governanceInsight](https://docs.microsoft.com/en-us/graph/api/resources/governanceInsight?view=graph-rest-beta) resource",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:678]:
```suggestion
              "Description": "Added the [governanceInsight](https://docs.microsoft.com/en-us/graph/api/resources/governanceInsight?view=graph-rest-beta) resource. This is an abstract type that exposes resources for configuring the type of insights to provide to decision-makers in an accessReviewDecisionItem.",
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 20]
```diff
@@ -627,6 +627,72 @@
...
+              "Target": "governanceInsight"
+            },
+            {
+              "Id": "e96e9533-1955-47a6-92ed-3e9a45e87e1f",
+              "ApiChange": "Resource",
+              "ChangedApiName": "userSignInInsight",
+              "ChangeType": "Addition",
+              "Description": "Added the [userSignInInsight](https://docs.microsoft.com/en-us/graph/api/resources/userSignInInsight?view=graph-rest-beta) resource",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:686]:
```suggestion
              "Description": "Added the [userSignInInsight](https://docs.microsoft.com/en-us/graph/api/resources/userSignInInsight?view=graph-rest-beta) resource. Use this type to configure insights for an **accessReviewDecisionItem** based on the user's last sign-in date and time.",
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 21]
```diff
@@ -627,6 +627,72 @@
...
+              "Target": "userSignInInsight"
+            }
+          ],
+          "Id": "e96e9533-1955-47a6-92ed-3e9a45e87e1f",
+          "Cloud": "Prod",
+          "Version": "beta",
+          "CreatedDateTime": "2021-10-25T22:11:38.5989437Z",
+          "WorkloadArea": "Microsoft.IdentityGovernance.AccessReviews",
+          "SubArea": ""
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:695]:
```suggestion
          "WorkloadArea": "Identity and access",
          "SubArea": "Governance"
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
@shubhamguptacal , left review comments here and there. Also,

- I notice that the instance relationship in accessReviewInstanceDecisionItem has never been published to the public metadata. Please look into this.
- Please move the enums and the changelog to first in their respective files
- For the **author** field, please change to Isabelle's alias. According to the guidance, "Use the GitHub name of the owner of the documentation for the workload. The owner is responsible for ensuring that external PRs are reviewed and approved and is assigned issues filed on the Docs site. ". The author must also be included in the Microsoft Graph contacts list.

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**